src/Http/Authentication.Abstractions/src/AuthenticateResult.cs

```diff
@@ -30,7 +30,7 @@ public class AuthenticateResult
     public AuthenticationTicket? Ticket { get; protected set; }
 
     /// <summary>
-    /// Gets the claims-principal with authenticated user identities.
```

Comment:
I think this is grammatically correct.

Think of it like this sentence:
"Gets the ice cream with chocolate syrup."

Noun - Ice cream (claims-principal) is what we're getting.
Adjective - Chocolate syrup (authenticated user identities) describes something about the noun.

"with which the" would make sense if the sentence ended with "identifies", but here we're specifically talking about the `ClaimsPrincipal.Identities` property for the authenticated user.

Reply:
You are right for some reason I read "identities" as "identifies"